engine/core/IgeMatrix2d.ts

```diff
@@ -427,7 +427,7 @@ export class IgeMatrix2d extends IgeBaseClass {
 	 */
 	transformRenderingContextSet (ctx: IgeCanvasRenderingContext2d) {
 		const m = this.matrix;
-		ctx.setTransform(m[0], m[3], m[1], m[4], m[2], m[5]);
+		ctx.setTransform(m[0], m[3], m[1], m[4], Math.floor(m[2]), Math.floor(m[5]));
```

Comment:
I'm a bit out of the loop on this change, what was the rationale here? Is it about avoiding sub-pixel rendering or something?